console-extensions.json

```diff
@@ -0,0 +1,69 @@
+[
+  {
+    "type": "console.flag/model",
+    "properties": {
+      "flag": "CRONTAB",
+      "model": {
+        "group": "stable.example.com",
+        "kind": "CronTab",
+        "version": "v1"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/resource-ns",
```

Comment:
We should make sure the nav extension is only enabled when the feature flag is set.